src/plugins/thirdParty/Python/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set(NUMERIC_LIBRARY_VERSION 37)
 
 # Git tag of our Python sources
 
-set(GIT_TAG "opencor-3.7.5")
+set(GIT_TAG "opencor")
```

Comment:
Why has the Git tag changed? I can't see it [here](https://github.com/opencor/python-cmake-buildsystem/tags).

Reply:
It is currently referring to the `opencor` branch that has another commit (after `opencor-3.7.5) that checks for and builds the `lzma` extension. Assuming everything tests OK then I'll tag the branch (with say `opencor-3.7.5.1`).

Maybe `GIT_REF` is a better name than `GIT_TAG`??

Reply:
To use `GIT_REF` would be confusing since it's used in our call to `ExternalProject_Add()` and that it's for the `GIT_TAG` parameter, hence the name indeed.

This aside, I now understand why `opencor` and I don't think it should be done that way. Instead, we should indeed have a new tag (e.g. `opencor-3.7.5.1` indeed). Then, once we know that everything works as expected, we should remove our current `opencor-3.7.5` tag and replace it with our new one.

Reply:
By the way, don't worry about it, I am going to do it. I am just finishing an issue with the CellML API (see issue #2240) and I will then do this one.